ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -447,6 +447,8 @@ presubmits:
         - ./ci-operator/jobs/
         - --plugin-config
         - ./core-services/prow/02_config/_plugins.yaml
+        - --supplemental-plugin-config-dir
+        - ./core-services/prow/02_config
```

Comment:
I think this is somehow broken

Reply:
For some reason the leading `./` break it

Reply:
Interesting ... same as

https://github.com/openshift/release/blob/ae90d9e55c4fae4627ff6d31d230f60e33458d29/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L444-L445

Dir is different from file? Maybe we need a / suffix as

https://github.com/openshift/release/blob/ae90d9e55c4fae4627ff6d31d230f60e33458d29/ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml#L446-L447